pkgs/development/python-modules/manuf/internal_db_update_nix.patch

```diff
@@ -0,0 +1,51 @@
+diff --git a/manuf/manuf.py b/manuf/manuf.py
+index e5e9193..0ac9296 100755
+--- a/manuf/manuf.py
++++ b/manuf/manuf.py
+@@ -61,12 +61,19 @@ class MacParser(object):
+         IOError: If manuf file could not be found.
+ 
+     """
+-    MANUF_URL = "https://gitlab.com/wireshark/wireshark/raw/master/manuf"
++    # https://github.com/coolbho3k/manuf/issues/34
++    MANUF_URL = "https://www.wireshark.org/download/automated/data/manuf"
+     WFA_URL = "https://gitlab.com/wireshark/wireshark/raw/master/wka"
+ 
+     def  __init__(self, manuf_name=None, update=False):
+-        self._manuf_name = manuf_name or self.get_packaged_manuf_file_path()
+-        if update:
++        if manuf_name is not None:
++            self._manuf_name = manuf_name
++            self.external_db = True
++        else:
++            self._manuf_name = self.get_packaged_manuf_file_path()
++            self.external_db = False
++
++        if update and self.external_db:
+             self.update()
+         else:
+             self.refresh()
+@@ -134,6 +141,9 @@ class MacParser(object):
+             URLError: If the download fails
+ 
+         """
++        if manuf_url is None and not self.external_db:
++            return
++
+         if not manuf_url:
+             manuf_url = self.MANUF_URL
+         if not manuf_name:
+diff --git a/manuf/test/test_manuf.py b/manuf/test/test_manuf.py
+index b60ffe2..c5816ff 100644
+--- a/manuf/test/test_manuf.py
++++ b/manuf/test/test_manuf.py
+@@ -8,7 +8,8 @@ class ManufTestCase(unittest.TestCase):
+ 
+     def setUp(self):
+         self.manuf = manuf.MacParser(manuf_name="manuf/test/manuf")
+-    #
++
++    @unittest.skip("Won't work on nix, requires internet access")
```

Comment:
We can use `disabledTests` with `pytestCheckHook`.

Reply:
Done